ext/nokogiri/extconf.rb

```diff
@@ -594,6 +594,9 @@ def do_clean
 append_ldflags(ENV["LDFLAGS"].split) unless ENV["LDFLAGS"].nil?
 $LIBS = concat_flags($LIBS, ENV["LIBS"])
 
+# c99 is old enough to drink
```

Comment:
True! Although it might be clearer to future readers to indicate why this is necessary. Either mention the use of C99 features or link to the GitHub issue?

Reply:
Done.

Reply:
Love the comment :-)